File: CodeWars/Sequence_Sum/sol.py
n = int(input())
ans = 0
for i in range(1, n + 1):
   if i % 5 == 0 and i % 3 == 0:
      ans += 0
   elif i % 3 == 0:
      ans += 3
   elif i % 5 == 0:
      ans += 5
   else:
      ans += i
print(ans)

File: CodeWars/Sequence_Sum/sol.cpp
#include <iostream>
using namespace std;

int main() {
   int n;
   cin >> n;
   int ans = 0;
   for (int i = 1; i <= n; ++i) {
      if (i % 5 == 0 and i % 3 == 0) {
         ans += 0;
      } else if (i % 3 == 0) {
         ans += 3;
      } else if (i % 5 == 0) {
         ans += 5;
      } else{
         ans += i;
      }
   }
   cout << ans << '\n';
}